pkg/server/plugin/bundlepublisher/gcpcloudstorage/gcpcloudstorage_test.go

```diff
@@ -0,0 +1,427 @@
+package gcpcloudstorage
+
+import (
+	"context"
+	"crypto/x509"
+	"errors"
+	"io"
+	"testing"
+
+	"cloud.google.com/go/storage"
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/spiffe/spire-plugin-sdk/pluginsdk/support/bundleformat"
+	bundlepublisherv1 "github.com/spiffe/spire-plugin-sdk/proto/spire/plugin/server/bundlepublisher/v1"
+	"github.com/spiffe/spire-plugin-sdk/proto/spire/plugin/types"
+	configv1 "github.com/spiffe/spire-plugin-sdk/proto/spire/service/common/config/v1"
+	"github.com/spiffe/spire/pkg/common/catalog"
+	"github.com/spiffe/spire/test/plugintest"
+	"github.com/spiffe/spire/test/spiretest"
+	"github.com/spiffe/spire/test/util"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/api/option"
+	"google.golang.org/grpc/codes"
+)
+
+var (
+	writeObjectCount int
```

Comment:
since packages are shared (code with test) can you rename this var to something like `testWriteObjectCount`?